gulpfile.js

```diff
@@ -1,3 +1,4 @@
+var _           = require('lodash');
```

Comment:
It must have come from a previous version, I'm going to remove it from gulp.js and its dependency on package.json